docs/dictionary/property/beepSound.lcdoc

```diff
@@ -25,6 +25,13 @@ set the beepSound to "internal"
 Example:
 set the beepSound to "vibrate"
 
+Example:
+-- on iOS there is no default beep sound, so you have to set one.
+on preopencard
```

Comment:
You need to add a `local tBeepPath` here otherwise LC will complain when in strict compilation mode

Reply:
Ah yes, sorry Andy, I forgot that requirement.